ui/components/multichain/import-tokens-modal/import-tokens-modal.js

```diff
@@ -0,0 +1,633 @@
+              className="import-tokens-modal__nft-address-error-link"
+              onClick={() => {
+                dispatch(showImportNftsModal());
+                onClose();
+              }}
+              color={TextColor.primaryDefault}
+              key="nftAddressError"
+            >
+              {t('importNFTPage')}
+            </ButtonLink>,
+          ]),
+        );
+        break;
+
+      case isMainnetToken && !isMainnet:
+        setMainnetTokenWarning(t('mainnetToken'));
+        setCustomSymbol('');
+        setCustomDecimals(0);
+        setCustomSymbolError(null);
+        setCustomDecimalsError(null);
+        break;
+
+      case Boolean(identities[standardAddress]):
+        setCustomAddressError(t('personalAddressDetected'));
+        break;
+
+      case checkExistingAddresses(address, tokens):
+        setCustomAddressError(t('tokenAlreadyAdded'));
+        break;
+
+      default:
+        if (!addressIsEmpty) {
+          attemptToAutoFillTokenParams(address);
+          if (standard) {
+            setTokenStandard(standard);
+          }
+        }
+    }
+  };
+
+  // Determines whether to show the Search/Import or Confirm action
+  const isConfirming = mode === 'confirm';
+
+  return (
+    <Modal
+      isOpen
+      onClose={() => {
+        dispatch(clearPendingTokens());
+        onClose();
+      }}
+      className="import-tokens-modal"
+    >
+      <ModalOverlay />
+      <ModalContent>
+        <ModalHeader
+          onBack={isConfirming ? () => setMode('') : null}
+          onClose={() => {
+            dispatch(clearPendingTokens());
+            onClose();
+          }}
+        >
+          {t('importTokensCamelCase')}
+        </ModalHeader>
+        {isConfirming ? (
+          <ImportTokensModalConfirm
+            onBackClick={() => {
+              dispatch(clearPendingTokens());
+              setMode('');
+            }}
+            onImportClick={async () => {
+              await handleAddTokens();
+              onClose();
+            }}
+          />
+        ) : (
+          <>
+            <Tabs t={t}>
+              {showSearchTab ? (
+                <Tab tabKey="search" name={t('search')}>
+                  <Box padding={4}>
+                    {useTokenDetection ? null : (
+                      <BannerAlert severity={Severity.Info} marginBottom={4}>
+                        <Text>
+                          {t('enhancedTokenDetectionAlertMessage', [
+                            networkName,
+                            <ButtonLink
+                              key="token-detection-announcement"
+                              onClick={() => {
+                                history.push(
+                                  `${SECURITY_ROUTE}#token-description`,
+                                );
+                                onClose();
+                              }}
+                            >
+                              {t('enableFromSettings')}
```

Comment:
Seems like this link is not working properly
https://github.com/MetaMask/metamask-extension/assets/39872794/8ef2b801-656a-446c-9d92-74a519062bd1